8.tcga-classify/classify-cancer-types.py

```diff
@@ -0,0 +1,234 @@
+"""
+2018 Gregory Way
+7.tcga-classify/classify-cancer-types.py
+
+Predict cancer-types using elastic net logistic regression and compressed gene
+expression features
+
```

Comment:
what different cancer types?  How many classes do you have? (maybe unnecessary to add, just curious)

Reply:
Is there a reason you chose to us elastic net?